### PR TITLE
Update top level README to match that at the EF

### PR DESCRIPTION
**Pull request by karianna**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,44 @@
...
+
+## Eclipse Adoptium Working Group
+
+The Adoptium Working Group promotes and supports high-quality runtimes and associated technology for use across the Java ecosystem.
+Our vision is to meet the needs of Eclipse and the broader Java community by providing a marketplace for high-quality Java runtimes for
+Java-based applications. We embrace existing standards and a wide variety of hardware and cloud platforms.
+
+## Eclipse Adoptium Top Level Project
+
+The mission of the Eclipse Adoptium Top-Level Project is to produce high-quality runtimes and associated technology for use within the Java ecosystem.  We achieve this through a set of Projects under the Adoptium PMC and a close working partnership with external projects, most notably OpenJDK for providing the Java SE runtime implementation. Our goal is to meet the needs of both the Eclipse community and broader runtime users by providing a comprehensive set of technologies around runtimes for Java applications that operate alongside existing standards, infrastructures, and cloud platforms.
```
Comment (sxa) [inline README.md:28]: Perhaps "distribute" rather then "produce" when we consider the marketplace?

Comment (karianna) [reply README.md]: Updated.

File: .github/ISSUE_TEMPLATE/release-checklist.md [hunk 2]
```diff
@@ -63,7 +65,7 @@ Release Week Checklist:
...
+- [ ] **Trigger docker images pipeline** via the [GitHub Actions job](https://github.com/AdoptOpenJDK/openjdk-docker/actions/workflows/updater.yml). Click Run Workflow, keep the branch as Master and click `Run Workflow`. This updater job checks the API and updates the relevant dockerfiles. A Pull Request is then created by the adoptopenjdk-bot account which will need merging once all the CI tests pass.  *TBD - how this step evolves for Temurin releases.
+- [ ] **Update Official Docker Images** Once the dockerfiles have been updated in the previous step you can run the `dockerhub_doc_config_update.sh` script. This will create a file called adoptopenjdk which will need to be copy and pasted into the [official adoptopenjdk config](https://github.com/docker-library/official-images/blob/master/library/adoptopenjdk) [example PR](https://github.com/docker-library/official-images/pull/10083) *TBD - how this step evolves for Temurin releases.
```
[the remark this answers is not in the document]
Comment (sxa) [reply .github/ISSUE_TEMPLATE/release-checklist.md:71]: Those comments are for the installers which are not used in the docker containers :-)

**Review by sxa — APPROVED** [review]
Other than the probably-out-of-date docker bits I'm ok with this., but I'd rather Shelley gave it a quick review especially regarding the definition of the Aqavit stuff before merging.

File: .github/ISSUE_TEMPLATE/release-checklist.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (karianna) [reply .github/ISSUE_TEMPLATE/release-checklist.md:71]: Oh right, I'll likely wait for George to update that when he comes back.

File: README.md [hunk 3]
```diff
@@ -1,11 +1,44 @@
...
+
+## Eclipse AQAvit project
+
+AQAvit is the quality and runtime branding evaluation project for Java SE runtimes and associated technology.  During a release it takes a functionally complete Java runtime and ensures that all the additional qualities are present that make it suitable for production use.  These quality criteria include good performance, exceptional security, resilience and endurance, and the ability to pass a wide variety of application test suites.  In addition to verifying that functionally complete runtimes are release ready, the AQA tests may also serve to verify new functionality during runtime development.
```
Comment (smlambert) [inline README.md:32]: Given some discussions in WG, we may want to update the overview of AQAvit at https://projects.eclipse.org/projects/adoptium.aqavit to reword 'branding' but agree that using that overview text here is what I would expect.

File: README.md [hunk 4]
```diff
@@ -1,11 +1,44 @@
...
+
+## Eclipse Temurin project
+
+The Eclipse Temurin™ project provides code and processes that support the building of runtime binaries and associated technologies that are high performance, enterprise-caliber, cross-platform, open-source licensed, and Java SE TCK-tested for general use across the Java ecosystem.
+
+## Eclipse Temurin Compliance project
+
+The Eclipse Temurin Compliance project is responsible for obtaining, managing, and executing the Oracle Java SE Compatibility Kit (JCK) on Eclipse Temurin binaries.  The work is done on private infrastructure and using code managed in closed repositories only available to committers of the Temurin Compliance.  The public artefacts produced by this project are limited to an indication of whether a particular Eclipse Temurin binary is Java SE compliant or not.
```
Comment (smlambert) [inline README.md:40]: yikes, caught me on the British spelling of artefacts (versus artifacts)!

**Review by smlambert — APPROVED** [review]
One minor comment, otherwise LGTM

**Pull request merged**